core/app.py

```diff
@@ -3,12 +3,16 @@
 """
 import connexion
 from config import Config
-from flask import Flask
+from flask import Blueprint, Flask, current_app
```

Comment:
Do Blueprint and current_app need to be imported here for the util to work? Is there any way to avoid that if so?

Reply:
Removed :)